spec/book_spec.rb

```diff
@@ -1,43 +1,43 @@
-require_relative '../classes/book'
-require 'date'
+# require_relative '../classes/book'
+# require 'date'
 
-describe Book do
-  before :each do
-    @book = Book.new('2023-01-01', 'Example Publisher', 'good', 'hello world')
-  end
+# describe Book do
+#   before :each do
+#     @book = Book.new('2023-01-01', 'Example Publisher', 'good', 'hello world')
+#   end
 
-  describe '#initialize' do
-    it 'sets the publish date, publisher, and cover state' do
-      expect(@book.publish_date).to eq('2023-01-01')
-      expect(@book.publisher).to eq('Example Publisher')
-      expect(@book.cover_state).to eq('good')
-    end
-  end
+#   describe '#initialize' do
+#     it 'sets the publish date, publisher, and cover state' do
+#       expect(@book.publish_date).to eq('2023-01-01')
+#       expect(@book.publisher).to eq('Example Publisher')
+#       expect(@book.cover_state).to eq('good')
+#     end
+#   end
 
-  describe '#move_to_archive' do
-    context 'when the cover state is bad' do
-      before do
-        @book = Book.new('2023-01-01', 'Example Publisher', 'bad', 'hello world')
-      end
+#   describe '#move_to_archive' do
+#     context 'when the cover state is bad' do
+#       before do
+#         @book = Book.new('2023-01-01', 'Example Publisher', 'bad', 'hello world')
+#       end
 
```

Comment:
I think it would be better if you don't overwrite or commented on the dev branch code that has already been approved by someone so that we are going to maintain the consistency of our projects.🥱🥱🥱